src/recommendations/configure.md

```diff
@@ -0,0 +1,58 @@
+---
+group: product-recommendations
+title: Configuring Recommendations
+---
+
+After you [install]({{ page.baseurl }}/recommendations/install.html) the `product-recommendations` module, you need to configure the catalog SaaS export module, set the environment, and specify the SaaS environment ID.
+
+## Configure the Catalog SaaS Export Module {#installcatalogsaas}
+
+The Catalog SaaS Export Module allows you to access your store's catalog data. You create a link to that data via an API key.
+
+{:.bs-callout-info}
+The Catalog SaaS export module is a requirement to successfully use the [Product Recommendations JS SDK](https://www.npmjs.com/package/@magento/recommendations-js-sdk). This module requires Magento 2.3 or later.
+
+### Provide API Key {#apikeys}
+
+1. Log in to your Magento account at `https://account.magento.com`.
+
+1. Under the **Magento** tab, select **API Portal** on the sidebar.
+
+1. Generate and copy the API key for **Production**.
+
+    ![]({{ page.baseurl }}/recommendations/images/get-api-key.png)
+
+{:.bs-callout-info}
```

Comment:
Indent this to align with the image. It's currently preventing the numbered list above from continuing below.